database.py

```diff
@@ -247,7 +270,7 @@ def __exit__(self, exc_type, exc_value, traceback):
             # returning False to let Python reraise the exception.
             return False
 
-    @attach_explain_query_plan
+    # @attach_explain_query_plan
```

Comment:
No, I will probably have to take a look at it and see if I can still let it do its job.